src/options.rs

```diff
@@ -27,4 +27,8 @@ pub struct Opt {
     /// Number of open files to set the ulimit to
     #[structopt(long, default_value = "50000")]
     pub open_files: u64,
+
+    /// Size in bytes of how much space the CPIO cahce is allowed to consume
+    #[structopt(long, default_value = "5 * 1024 * 1024 * 1024")]
```

Comment:
Can't do fancy maffs like this, and also can't make it be "raw" and have the compiler calculate it :(

```suggestion
    #[structopt(long, default_value = "5368709120")] // 5 GiB in bytes
```